script/cileaks.sh

```diff
@@ -9,5 +9,5 @@ then
 fi
 
 if [ -n "$VALGRIND" -a -e "$(which valgrind)" ]; then
-	valgrind --leak-check=full --show-reachable=yes --error-exitcode=125 --suppressions=./libgit2_clar.supp $@ _build/libgit2_clar -ionline
+	valgrind --leak-check=full --show-reachable=yes --error-exitcode=125 --suppressions=./libgit2_clar.supp $@ _build/libgit2_clar -ionline -xbuf::oom
```

Comment:
Is this test still useful at all?  Maybe we should remove it entirely.

Reply:
I was originally planning to try and make leaks works too, but I can remove it in the mean time.